src/api/helpers/get-populated-application/index.test.ts

```diff
@@ -69,6 +72,27 @@ describe('api/helpers/get-populated-application', () => {
     expect(result.buyer.country?.isoCode).toEqual(expectedCountry.isoCode);
   });
 
+  it('should return an application with populated nominatedLossPayee', async () => {
+    const result = await getPopulatedApplication(context, applicationIds);
+
+    expect(result.nominatedLossPayee.isAppointed).toEqual(mockNominatedLossPayee.isAppointed);
+    expect(result.nominatedLossPayee.isLocatedInUk).toEqual(mockNominatedLossPayee.isLocatedInUk);
+    expect(result.nominatedLossPayee.isLocatedInternationally).toEqual(mockNominatedLossPayee.isLocatedInternationally);
+    expect(result.nominatedLossPayee.name).toEqual(mockNominatedLossPayee.name);
+  });
+
+  it('should return an application with populated financialUk', async () => {
+    const result = await getPopulatedApplication(context, applicationIds);
+
+    const { financialUk } = result.nominatedLossPayee;
+
+    expect(financialUk.accountNumber).toEqual(mockLossPayeeFinancialDetailsUk.accountNumber);
+    expect(financialUk.accountNumberVector).toEqual(mockLossPayeeFinancialDetailsUk.accountNumberVector);
+    expect(financialUk.sortCode).toEqual(mockLossPayeeFinancialDetailsUk.sortCode);
+    expect(financialUk.sortCodeVector).toEqual(mockLossPayeeFinancialDetailsUk.sortCodeVector);
+    expect(financialUk.bankAddress).toEqual(mockLossPayeeFinancialDetailsUk.bankAddress);
+  });
+
```

Comment:
Can we add an error handling test please like for the other pieces of logic, 

`it('should throw an error`